lib/elixir/lib/kernel.ex

```diff
@@ -3401,6 +3401,45 @@ defmodule Kernel do
   In the example above, a `sum/2` function is defined; this function receives
   two arguments and returns their sum.
 
+  ## Default arguments
+
+  `\\` is used to specify a default value for a parameter of a function. For
+  example:
+
+      defmodule Foo do
+        def mul_by(x, n \\ 2) do
+          x * n
+        end
+      end
+
+      Foo.mul_by 4, 3 #=> 12
+      Foo.mul_by 4    #=> 8
+
+  The compiler translates this into multiple function clauses with different
```

Comment:
I'm not sure we should call these "clauses". Usually "clauses" refers to the multiple clauses of a single function, but functions with different arities are properly different functions. So we could say "The compiler translates this into multiple functions with different arities ...".